fili-core/src/main/java/com/yahoo/bard/webservice/web/TableFullViewProcessor.java

```diff
@@ -95,14 +98,44 @@ public TableGrainView formatTableGrain(LogicalTable logicalTable, String grain,
         resultRow.put("longName", StringUtils.capitalize(grain));
         resultRow.put("description", "The " + logicalTable.getName() + " " + grain + " grain");
         resultRow.put("retention", logicalTable.getRetention().toString());
-        resultRow.put(
-                "dimensions",
-                DimensionsServlet.getDimensionListSummaryView(logicalTable.getDimensions(), uriInfo)
-        );
+        resultRow.put("dimensions", getDimensionListFullView(logicalTable.getDimensions(), uriInfo));
```

Comment:
The reason the change is here, rather than in the dim servlet, is to keep the change to the external API as small as possible. If we change it in the dim servlet, then we end up changing the dimension view as well, which I didn't want to do.

What _really_ needs to happen is we need to put an abstraction in place so that these formats can be their own extensions and are pluggable, but I didn't want to tackle that in this PR.